src/main/java/com/volasoftware/tinder/controller/UserController.java

```diff
@@ -19,11 +19,19 @@ public UserController(UserService userService) {
         this.userService = userService;
     }
 
-    @GetMapping("/api/v1/users/profile")
+    @GetMapping("/api/v1/users/show-profile")
```

Comment:
Read more about: https://stackoverflow.blog/2020/03/02/best-practices-for-rest-api-design/